root/communication-rmi/src/main/java/at/fhv/itb5c/rmi/server/RMIServer.java

```diff
@@ -0,0 +1,26 @@
+package at.fhv.itb5c.rmi.server;
+
+import java.net.MalformedURLException;
+import java.rmi.RemoteException;
+import java.util.List;
+
+public class RMIServer {
+	private static List<RMIServant> _servants;
+	private static final String _host = "localhost";
+	private static final int _port = 1337;

```

Comment:
Move these settings to a [properties](https://docs.oracle.com/javase/tutorial/essential/environment/properties.html) file.